src/main.js

```diff
@@ -74,7 +66,7 @@ function formVisibility() {
   savedViewSection.classList.add("hidden");
 };
 
-function savedPosterGridView() {
+function savedCoverGridView() {
```

Comment:
There seems to be a lot of adding and remove the "hidden" attribute from classes to show pages. It works great the way you have it, but this repetition makes me wonder if there's a good opportunity to refactor a few functions into one collecting changePageView function using a parameter to establish which page to display.